modules/workspace.nix

```diff
@@ -99,14 +108,27 @@ in
         Allows you to set wallpaper slideshow. Needs a directory of your wallpapers and an interval length.
       '';
     };
+
+    wallpaperPictureOfTheDay = lib.mkOption {
+      type = lib.types.nullOr wallpaperPictureOfTheDayType;
+      default = null;
+      example = ''{ provider = "apod" }'';
```

Comment:
Should be `{ provider = "apod"; }` instead (i.e. remove the quotes and add semicolon after provider).

Reply:
Well, POTD seems to have another option to enable/disable updating over a metered connection. Adding it immediately.
> Should be `{ provider = "apod"; }` instead (i.e. remove the quotes and add semicolon after provider).
Even after being told that the example was not supposed to be a string, I fixed it...and changed it to another string. :sweat_smile: